.github/actions/install-dashboards/action.yml

```diff
@@ -68,14 +64,18 @@ runs:
         node-version: ${{ steps.tool-versions.outputs.node_version }}
         registry-url: 'https://registry.npmjs.org'
 
-    - run: |
+    - name: Setup Opensearch Dashboards
+      run: |
         npm uninstall -g yarn
         echo "Installing yarn ${{ steps.tool-versions.outputs.yarn_version }}"
         npm i -g yarn@${{ steps.tool-versions.outputs.yarn_version }}
+        yarn cache clean
+        yarn add sha.js
```

Comment:
What does `yarn add sha.js` do?

Reply:
@cwperx The reason why we clear the cache for that is I notice that we got a warn for yarn if we only run the setup commands for yarn version, so I use the clear cache command and add sha.js to fix the yarn version warning [1].

Reference: 
[1] : https://github.com/crypto-browserify/sha.js/issues/64#issuecomment-567987159